compiler/tflchef/core/src/CustomOp/MaxPoolWithArgMax.cpp

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) 2020 Samsung Electronics Co., Ltd. All Rights Reserved
+ * Copyright 2015 The TensorFlow Authors. All Rights Reserved.
```

Comment:
What code is from TensorFlow?

Reply:
Please look at this again

Reply:
I think only `REGISTER_OP("MaxPoolWithArgmax")` comment is taken from original TF codebase.
Should we remove this line?

Reply:
Ah, OK, I think we need this comment :)